reference/6/Microsoft.PowerShell.Core/About/about_Signing.md

```diff
@@ -139,7 +139,7 @@ topic.
 
 ## Create a self-signed certificate
 
-To create a self-signed certificate in use the `New-SelfSignedCertificate`
+To create a self-signed certificate in PowerShell 3.0 or later, use the `New-SelfSignedCertificate`
```

Comment:
I think we could use the reference to the module being introduced in 3.0  as enough and shorten this one.
```suggestion
To create a self-signed certificate use the `New-SelfSignedCertificate`
```